skimage/morphology/tests/test_watershed.py

```diff
@@ -483,5 +483,11 @@ def test_numeric_seed_watershed():
     np.testing.assert_equal(compact, expected)
 
 
+def test_water(arr1, arr2):
```

Comment:
@JohnnyTeutonic can you please rename this to something more meaningful? e.g. `test_incorrect_markers_shape`. Also, test functions don't take any arguments, which is why the Travis build is failing. ;)